src/main/resources/application.properties

```diff
@@ -14,6 +14,9 @@ spring.security.oauth2.client.registration.google.client-id=${GOOGLE_CLIENT_ID:$
 spring.security.oauth2.client.registration.google.client-secret=${GOOGLE_CLIENT_SECRET:${env.GOOGLE_CLIENT_SECRET:client_secret_unset}}
 spring.security.oauth2.client.registration.google.scope=email,profile
 
+#helps MONGODB stuff
+spring.data.mongodb.database=cs156-6pm-2-courses-project
+
```

Comment:
A couple of things:

* It's not generally a good idea to combine unrelated things in the same PR.  
* I'm confused as to why this is needed.   What problem are you trying to solve with this?
* If we do need this, it should be located next to the other MONGODB related stuff on line 28.
* If we do need this, it should be set up similar to what is on line 28, so that it can be overridden with config variables, i.e. 

```
spring.data.mongodb.database=${MONGODB_DB:${env.MONGODB_DB:cs156-6pm-2-courses-project}
```

But, I think that in fact, we are going to solve this problem when we distribute mongodb credentials and give you real values for the MONGODB_URI in a couple of days. 

In any case, I would like to encourage you to make this a separate PR and a separate issue, and NOT include it in this one.

Reply:
Understood, I reverted the application.properties file to what is was originally so that this PR is solely for the "Fix Brand" issue.